lsp/vscode/src/extension.ts

```
@@ -1,52 +1,128 @@
 import type { InitializationOptions } from '@volar/language-server'
 import * as serverProtocol from '@volar/language-server/protocol'
-import { type ExportsInfoForLabs, activateAutoInsertion, supportLabsVersion } from '@volar/vscode'
+import { activateAutoInsertion, supportLabsVersion, type ExportsInfoForLabs } from '@volar/vscode'
+import * as path from 'path'
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)